src/main/java/com/scalar/admin/AdminClient.java

```diff
@@ -52,6 +52,15 @@ public Optional<String> stats() {
     }
   }
 
+  public Optional<String> paused() {
```

Comment:
I used Optional<String> as the return type to consistent with the stats() function so the RequestCoordinator can use the same `run` logic.